### ISSUE TITLE
[bug]: DLL load failed while importing onnx_cpp2py_export

### ISSUE THREAD

**Issue by TobiasReich** [invoke-ai/InvokeAI#7111]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

RTX 4080

### GPU VRAM

16

### Version number

5.2.0rc1

### Browser

Firefox

### What happened

I had some dependency issues (newer ones were installed during installation and thus I could not proceed with the installation). The latest version 5.2.0rc1 solved this.
I installed invokeAI the usual way but when I started it I got this error.

> Desired action:
> 1. Generate images with the browser-based interface
> 2. Open the developer console
> 3. Command-line help
> Q - Quit
> 
> To update, download and run the installer from https://github.com/invoke-ai/InvokeAI/releases/latest
> 
> Please enter 1-4, Q: [1]
> Starting the InvokeAI browser-based UI..
> Traceback (most recent call last):
>   File "C:\Users\T\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
>     return _run_code(code, main_globals, None,
>   File "C:\Users\T\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
>     exec(code, run_globals)
>   File "C:\Users\T\invokeai\.venv\Scripts\invokeai-web.exe\__main__.py", line 7, in <module>
>     sys.exit(run_app())
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\app\run_app.py", line 10, in run_app
>     from invokeai.app.api_app import invoke_api
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\app\api_app.py", line 24, in <module>
>     from invokeai.app.api.dependencies import ApiDependencies
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\app\api\dependencies.py", line 14, in <module>
>     from invokeai.app.services.download.download_default import DownloadQueueService
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\app\services\download\__init__.py", line 3, in <module>
>     from invokeai.app.services.download.download_base import (
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\app\services\download\download_base.py", line 13, in <module>
>     from invokeai.backend.model_manager.metadata import RemoteModelFile
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\backend\model_manager\__init__.py", line 16, in <module>
>     from invokeai.backend.model_manager.load import LoadedModel
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\backend\model_manager\load\__init__.py", line 9, in <module>
>     from invokeai.backend.model_manager.load.load_base import LoadedModel, LoadedModelWithoutConfig, ModelLoaderBase
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\backend\model_manager\load\load_base.py", line 21, in <module>
>     from invokeai.backend.model_manager.load.model_cache.model_cache_base import ModelCacheBase, ModelLockerBase
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\backend\model_manager\load\model_cache\__init__.py", line 4, in <module>
>     from .model_cache_default import ModelCache  # noqa F401
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\backend\model_manager\load\model_cache\model_cache_default.py", line 23, in <module>
>     from invokeai.backend.model_manager.load.model_util import calc_model_size_by_data
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\backend\model_manager\load\model_util.py", line 20, in <module>
>     from invokeai.backend.onnx.onnx_runtime import IAIOnnxRuntimeModel
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\invokeai\backend\onnx\onnx_runtime.py", line 8, in <module>
>     import onnx
>   File "C:\Users\T\invokeai\.venv\lib\site-packages\onnx\__init__.py", line 77, in <module>
>     from onnx.onnx_cpp2py_export import ONNX_ML
> ImportError: DLL load failed while importing onnx_cpp2py_export: A dynamic link library (DLL) initialization routine failed.

I did the usual fixes like
- a fresh install of the Geforce Drivers.
- calling the installer again.
- I tried to delete the .venv folder in the invoke root folder.
- And even  tried to install the dependency manually with no change.

Any idea what I might try next?

### What you expected to happen

The application should start as usual.

**Comment by BeastDD** [comment]
There is currently an issue between ONNX and ONNX runtime on the latest version of InvokeAI. 
For some reason when I use the InvokeAI installer, after updating it my InvokeAI installation I realized that it doesn't create all the folder it used to.  I also tried it on a random new folder installation to trigger a full new install, and it did the same thing once again. It just created a .venv folder but none of the models, config, database and so on little folders that it used to make... It just generate or update the .venv folder and it **downloads incompatible versions of ONNX and ONNX Runtime**.
You're the lucky though as I pieced a solution together after looking into forums  and running some test.

1. Update the Invoke AI installer
2. Pick a place to install the InvokeAI app. (it can be new or an old one  that you are updating)
3. Launch InvokeAI and pick option 2 the "Developer Console"
4. Now enter this:
`pip uninstall onnx onnxruntime` 

Press (Y) to both time that it ask you if you want to uninstall the respective packages
Then enter this next 

 `pip install onnx==1.16.1 onnxruntime==1.16.3`

Let it do its thing and write "exit"

5. Proceed to choose option 1  - Your InvokeAI should launch no problems now.

TLDR - Uninstall the version of ONNX and ONNX Runtime that you have currently install in your venv and reinstall ONNX 1.16.1 & ONNX Runtime 1.16.3

_Note:_ 

_- These are the earliest version of ONNX and ONNX Runtime I could run together with my InvokeAI installation. If you try >1.16.2 for ONNX and >1.16.3ONNX Runtime you will constantly run into this error._

_- This behavior has been happening to me since the previous update of 5.2.0rc1 and is currently still happening with 5.2.0rc1. I don't know what cause this but ONNX and ONXX runtime gets updated to their latest respective version when updating InvokeAI and both of these versions seem to not be working for InvokeAI._

>  DLL load failed while importing onnx_cpp2py_export: A dynamic link library (DLL) initialization routine failed.
